src/components/MarkdownLayout/MarkdownLayout.tsx

```diff
@@ -119,8 +119,8 @@ export default function MarkdownLayout({
   //   document.querySelector('html').style.scrollBehavior = 'smooth';
   //   return () => (document.querySelector('html').style.scrollBehavior = 'auto');
   // }, []);
-  const problemSolutionContext = React.useContext(ProblemSolutionContext);
-  let activeIDs = [];
+  const problemSolutionContext = useProblemSolutions();
+  let activeIDs: string[] = [];
```

Comment:
`useProblemSolutions()` throws an error if `ProblemSolutionContext` does not exist. `ProblemSolutionContext` exists only for solution pages, but `MarkdownLayout` is rendered both for solution pages and module pages (this should maybe be refactored).

`React.useContext()` returns null / undefined if `ProblemSolutionContext` does not exist, which is what we want here. We can assert that `problemSolutionContext` is defined if `markdownData` is not an instance of `ModuleInfo`.